python/scripts/remove-collaborator-locked-repos.py

```diff
@@ -129,19 +129,19 @@ def fetch_repositories(github_gql_client: Client, github_api_client: Github) ->
     return repositories_list
 
 
-def remove_collaborator(collaborator, github_api_client: Github):
+def remove_collaborator(collaborator, github_service: GithubService):
     """Remove the collaborator from the organisation
     Args:
         collaborator (collaborator): The collaborator object
     """
     print("Remove user from organisation: " + collaborator.login)
-    org = github_api_client.get_organization("ministryofjustice")
-    org.remove_outside_collaborator(collaborator)
```

Comment:
This line has been removed temporarily whilst we're refactoring to enable testing 🧪

I'll add it back in once the code has been refactored with enough unit tests to ensure changes don't break functionality